CODE_OF_CONDUCT.md

```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at gregor.sturm@i-med.ac.at. All
+reported by contacting the project team at steering-committee@omnideconv.org. All
```

Comment:
@mlist, do we have an "official" omnideconv email address? Or what address should I list here?